Add French translation (#74)
organicmaps/organicmaps.github.io: content/_index.fr.md

---
title: "Organic Maps: cartes hors ligne et navigateur GPS"
description: Cartes open source pour voyageurs, touristes, cyclistes et randonneurs.
page_template: index.html
sort_by: weight
extra:
  menu_title: Page d'accueil
---

> **Organic Maps** est une application Android et iOS gratuite de cartes hors ligne pour les voyageurs, les touristes, les randonneurs et les cyclistes, basée sur des données provenant de la foule. **[OpenStreetMap](https://www.openstreetmap.org)**. Il s'agit d'un [fork](https://en.wikipedia.org/wiki/Fork_(software_development)) de l'application **Maps.me** (précédemment connue sous le nom de [**MapsWithMe**](https://en.wikipedia.org/wiki/Maps.me)), axé sur la confidentialité et maintenu par les mêmes personnes qui ont créé **MapsWithMe** en 2011.

Reviewer: ```suggestion
**Organic Maps** est une application Android et iOS gratuite de cartes hors ligne pour les voyageurs, les touristes, les randonneurs et les cyclistes, basée sur des données provenant des contributeurs. **[OpenStreetMap](https://www.openstreetmap.org)**. Il s'agit d'un [fork](https://en.wikipedia.org/wiki/Fork_(software_development)) de l'application **Maps.me** (précédemment connue sous le nom de [**MapsWithMe**](https://en.wikipedia.org/wiki/Maps.me)), axé sur la confidentialité et maintenu par les mêmes personnes qui ont créé **MapsWithMe** en 2011.
```

- Pas de frais de données mobiles inattendus

L'application Organic Maps est exempte de traceurs et d'autres éléments indésirables :

• Une expérience hors ligne ultra-rapide
• Respecte votre vie privée
• Économise votre batterie
• Pas de frais de données mobiles inattendus
• Simplicité d'utilisation, avec uniquement les fonctions les plus importantes.

Sans trackers et autres mauvaises choses :

• Pas de publicité
• Pas de pistage
• Pas de collecte de données
• Pas d'appel au domicile
• Pas d'inscription fastidieuse
• Pas de tutoriels obligatoires
• Pas de spam excessif
• Pas de notifications push
• Pas de logiciels malveillants
• N̶o̶ ̶p̶e̶s̶t̶i̶c̶i̶d̶e̶s̶ Entièrement bio

Selon <a href='https://reports.exodus-privacy.eu.org/en/reports/app.organicmaps/latest/'>Exodus Privacy Project</a> il n'y a aucun traceur et les autorisations requises sont minimales:
<br/>
<img src='/images/privacy/exodus.png' width='400'>

Organic Maps ne demande pas de permissions excessives pour vous espionner:

<img src='/images/privacy/om.jpg' width='400'>
<img src='/images/privacy/mm.jpg' width='400'>

Chez Organic Maps, nous pensons que la <b>vie privée est un droit humain fondamental</b> :

> • Organic Maps est un projet indépendant à code source ouvert dirigé par la communauté.

Reviewer: ```suggestion
• Organic Maps est un projet indépendant open-source dirigé par la communauté.
```

• Nous protégeons la vie privée des regards indiscrets des géants de la tech
• Restez en sécurité, où que vous soyez

Reject surveillance - embrace your freedom.

<a href="#install"><strong>Essayez Organic Maps !</strong></a>

## Qui paie pour l'application gratuite ?

L'application est gratuite pour tous. Veuillez [faire un don](@/donate/index.md) pour nous soutenir !

### Il y a plusieurs façons de faire un don :

{{ donate_buttons(lang='fr') }}

### Nos sponsors :

> Le FAI [Mythic Beasts](https://www.mythic-beasts.com/) [nous](https://www.mythic-beasts.com/blog/2021/10/06/improving-the-world-bit-by-expensive-bit/) fournit deux serveurs virtuels avec 400 TB/mois de bande passante gratuite pour aider nos utilisateurs à téléchargements et mises à jour de cartes.

Reviewer: ```suggestion
Le FAI [Mythic Beasts](https://www.mythic-beasts.com/) [nous](https://www.mythic-beasts.com/blog/2021/10/06/improving-the-world-bit-by-expensive-bit/) fournit deux serveurs virtuels avec 400 TB/mois de bande passante gratuite pour aider nos utilisateurs à télécharger et mettre à jour les cartes.
```

**Organic Maps** est l'une des rares applications actuelles à prendre en charge 100 % des fonctionnalités sans connexion Internet active. Installez Organic Maps, téléchargez des cartes, jetez votre carte SIM (d'ailleurs, votre opérateur vous suit constamment), et partez pour un voyage hebdomadaire sur une seule charge de batterie sans aucun octet envoyé au réseau.

{{ badges(lang='fr') }}

{{ screenshot(src='/images/screenshots/hiking.jpg', alt='Hiking') }}
{{ screenshot(src='/images/screenshots/prague.jpg', alt='Prague') }}
{{ screenshot(src='/images/screenshots/search.jpg', alt='Offline Search') }}
{{ screenshot(src='/images/screenshots/dark.jpg', alt='Navigation in dark mode') }}

> ## Features

Reviewer: ```suggestion
## Fonctionnalités
```

Organic Maps est l'application idéale pour les voyageurs, les touristes, les randonneurs et les cyclistes :

- Cartes détaillées hors ligne avec des lieux qui n'existent pas sur d'autres cartes, grâce à [OpenStreetMap](https://osm.org)
- Itinéraires cyclables, sentiers de randonnée et chemins de promenade.
- Courbes de niveau, profils d'élévation, sommets et pentes.
- Navigation à pied, à vélo et en voiture, virage par virage, avec guidage vocal.
- Recherche rapide hors ligne sur la carte
- Exportation et importation de signets aux formats KML/KMZ (GPX est [prévu](https://github.com/organicmaps/organicmaps/issues/624))
- Mode sombre pour protéger vos yeux
- Les pays et les régions ne prennent pas beaucoup de place
- Gratuit et open-source

> ## Why Organic?

Reviewer: ```suggestion
## Pourquoi Organic?
```

Organic Maps est pur et biologique, fabriqué avec amour :

- Respecte votre vie privée
- Économise votre batterie
- Pas de frais de données mobiles inattendus

> L'application Organic Maps est exempte de traceurs et d'autres éléments indésirables :

Reviewer: ```suggestion
L'application Organic Maps est exempte de trackers et d'autres éléments indésirables :
```

- Une expérience hors ligne ultra-rapide
- Respecte votre vie privée
- Économise votre batterie
- Pas de frais de données mobiles inattendus
- Simplicité d'utilisation, avec uniquement les fonctions les plus importantes.

Sans trackers et autres mauvaises choses :

- Pas de publicité
- Pas de pistage
- Pas de collecte de données
- Pas d'appel au domicile
- Pas d'inscription fastidieuse
- Pas de tutoriels obligatoires
- Pas de spam excessif
- Pas de notifications push
- Pas de logiciels malveillants
- N̶o̶ ̶p̶e̶s̶t̶i̶c̶i̶d̶e̶s̶ Entièrement bio

> Selon <a href='https://reports.exodus-privacy.eu.org/en/reports/app.organicmaps/latest/'>Exodus Privacy Project</a> il n'y a aucun traceur et les autorisations requises sont minimales:

Reviewer: ```suggestion
Selon <a href='https://reports.exodus-privacy.eu.org/en/reports/app.organicmaps/latest/'>Exodus Privacy Project</a> il n'y a aucun tracker et les autorisations requises sont minimales:
```

<br/>
<img src='/images/privacy/exodus.png' width='400'>

Organic Maps ne demande pas de permissions excessives pour vous espionner:

<img src='/images/privacy/om.jpg' width='400'>
<img src='/images/privacy/mm.jpg' width='400'>

Chez Organic Maps, nous pensons que la <b>vie privée est un droit humain fondamental</b> :

• Organic Maps est un projet indépendant open-source dirigé par la communauté.
• Nous protégeons la vie privée des regards indiscrets des géants de la tech
• Restez en sécurité, où que vous soyez

> Reject surveillance - embrace your freedom.

Reviewer: ```suggestion
Rejeter la surveillance - embrasser votre liberté.
```
